ansible/roles/k8s_cluster/tasks/remove.yml

```diff
@@ -13,3 +13,6 @@
       - kubectl-{{k8s_version}}
       - kubelet-{{k8s_version}}
   become: yes
+
+- name: "Remove containerd"
+  include_tasks: "remove_containerd.yml"
```

Comment:
I'm not sure that we should remove it like docker daemon as we assume that to install a k8s cluster such environment exist within the VM

Reply:
I'm not sure if it's better than docker, I tested this after reading the news about *"Kubernetes is deprecating Docker as a container runtime after v1.20"*. Moreover, Docker gave me a hard time with Fedora. 

Why containerd and not CRI-O? I first tried installing CRI-O but wasn't successful, then I tried containerd and it went through.